### DISCUSSION TITLE
gltf-transform cli resize is failed.

### DISCUSSION THREAD

**Discussion by neciszhang** [donmccurdy/glTF-Transform#1097] [Q&A] [answered]
Hi,
`gltf-transform resize in.glb out.glb --width 512 --height 512` is failed.Nothing has changed
 gltf-transform cli version: 3.4.0.

**Comment by donmccurdy** [comment]
Hi @neciszhang, you'll probably need to share more information to get help with this. Some ideas:

1. Use `--verbose` to see any errors or warnings, and share them here.
2. Check the original sizes of the textures. Resizing should make sure they do not *exceed* 512x512, but it doesn't scale textures up or change their aspect ratio.
3. If neither of those options work, please share the model.

**Comment by neciszhang** [comment]
Hi,
![image](https://github.com/donmccurdy/glTF-Transform/assets/16221107/615ab26e-93a7-424d-b8c9-1bcdf7bb7230)
[here is model link](https://nft.elemecdn.com/test/in.glb)
I use [gltf.report](https://gltf.report/) to resize, this website can convert it successfully.

**Reply by donmccurdy** [reply, marked as answer]
I tested this model in glTF Transform v3.7.0, and that does reduce the resolution – could you try upgrading your version? Seems like it may be a bug in the old version.

You may also find that the file size (in bytes) increases after resizing, because the default png compression is lossless. If you'd like to apply lossy compression after resizing, you can do so with...

```bash
gltf-transform png in.glb out.glb --quality 80 --effort 100
```

... and the file size on disk will decrease as well.

**Reply by neciszhang** [reply]
Yes, when I upgraded to 3.7.0, the resize can be successful. I have another problem. When I use the etc1s ` gltf-transform etc1s in.glb out.glb,` I am prompted to spawn Unknown system error -86.

**Reply by donmccurdy** [reply]
I'm not sure what would cause that – was it working in the previous version? If you run `ktx -h` in a commandline what do you see? It might be best to file an issue for this at https://github.com/donmccurdy/glTF-Transform/issues/new/choose.

Thanks!

**Reply by neciszhang** [reply]
ok, it’s a problem with my software installation. I installed a new version of KTX-Software, but it doesn’t match my computer.

**Reply by donmccurdy** [reply]
Hmm, ok! I believe glTF Transform supports the v4.2 stable releases of KTX Software, but I'm not sure about the v4.3 alpha release yet.
